### ISSUE TITLE
Design Files

### ISSUE THREAD

**Issue by vinceprofeta** [zendeskgarden/react-components#288]
## Expectations

Is there a way to get access to sketch, illustrator, etc files? We would love to be able to give our designers the components we want to use to build out user flows!

## Reality

I cant find any online through the Zendesk design site.

## Steps to Reproduce

n/a

-->

### Fine Print

* Component: 
* Version: 
* Browsers: all

**Comment by allisonacs** [comment]
Hi @vinceprofeta! So awesome to see community interest in getting design assets. Unfortunately, we're not able to open source our design files right now. We use Abstract to manage our Sketch workflows. Although I hope that they will add open source support soon, they don't currently have it. Other solutions, like statically hosted files, are difficult to maintain for our small-but-mighty team.

TLDR: we can't do this right now, but hope that we will be able to do it in the future! Thanks for requesting it publicly, it helps quantify the demand. Cheers!
